codechat_config_cal_custom.yaml

```diff
@@ -0,0 +1,4 @@
+# Replace the contents of `codechat_config.yaml` with this file to edit Calculus
+output_path: output/cal-web-custom
+args: "{sys_executable} -m pretext build cal-web-custom --xmlid {xml_id} --no-knowls"
+project_type: PreTeXt
```

Comment:
I think these custom codechat config files should get your codechat working - don't forget to paste them into the main codechat file when working on one book vs another...